src/cc/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ set_target_properties(bcc-static PROPERTIES OUTPUT_NAME bcc)
 # BPF is still experimental otherwise it should be available
 #llvm_map_components_to_libnames(llvm_libs bpf mcjit irreader passes)
 llvm_map_components_to_libnames(llvm_libs bitwriter bpfcodegen irreader linker
-  mcjit objcarcopts option passes x86codegen)
+  mcjit objcarcopts option passes x86codegen powerpccodegen)
```

Comment:
that's correct for .so, but .a will include it all. Eventually fully static bcc-lua should stay the same though. Right.
The other concern is that some distros may not ship powerpc backend by default and the build will break on x86 though ppc backend is unused.
Can we do an 'if' inside cmake to include x86codegen or ppccodegen?